### ISSUE TITLE
Cannot compile with gcc now

### ISSUE THREAD

**Issue by newfrenchy83** [wesnoth/wesnoth#8107]
### Game and System Information

- **Version:**1.17 after https://github.com/wesnoth/wesnoth/commit/d22b70eda8a386df26a55046679a9e653e7d2107 commit
- **Downloaded from:**build from source
- **Build info:**mingw12.2
- **OS:**windows10

### Description of the bug

when i try to compile i this message
[log.txt](https://github.com/wesnoth/wesnoth/files/13636401/log.txt)

a bibrarie must be defined but i don't know what libraire needed.

**Comment by Pentarctagon** [comment]
Can you bisect this to see when it started happening?

**Comment by newfrenchy83** [comment]
> Can you bisect this to see when it started happening? The error doesn't seem to reference any library function.

the error begin with https://github.com/wesnoth/wesnoth/commit/d22b70eda8a386df26a55046679a9e653e7d2107

**Comment by Pentarctagon** [comment]
Did you update your mingw version recently?

**Comment by newfrenchy83** [comment]
[build-info.txt](https://github.com/wesnoth/wesnoth/files/13638132/build-info.txt) the log of mingw what icurrently using

**Comment by Pentarctagon** [comment]
I mean did *you* update it recently? Since none of the CI jobs are failing for this and nothing jumps out to me in that commit which would affect GUI classes while linking the executable.

**Comment by newfrenchy83** [comment]
no, i use version of a year ago x86_64-12.2.0-release-posix-seh-rt_v10-rev1

**Comment by newfrenchy83** [comment]
i used more recent and it's work

**Comment by Pentarctagon** [comment]
So you updated to a new version and it builds now?

**Comment by newfrenchy83** [comment]
> So you updated to a new version and it builds now?

Yes, i close this issue now.
